staging/src/k8s.io/kube-aggregator/pkg/registry/apiservice/rest/storage_apiservice.go

```diff
@@ -34,21 +34,27 @@ import (
 func NewRESTStorage(apiResourceConfigSource serverstorage.APIResourceConfigSource, restOptionsGetter generic.RESTOptionsGetter) genericapiserver.APIGroupInfo {
 	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(apiregistration.GroupName, aggregatorscheme.Registry, aggregatorscheme.Scheme, metav1.ParameterCodec, aggregatorscheme.Codecs)
 
-	if apiResourceConfigSource.VersionEnabled(v1beta1.SchemeGroupVersion) {
+	if apiResourceConfigSource.AnyResourcesForVersionEnabled(v1beta1.SchemeGroupVersion) {
```

Comment:
could this stay as it is because we don't want resource-based enablement for the aggregator?

Reply:
ok, missed this